src/celeritas/track/SimParams.hh

```diff
@@ -42,6 +42,7 @@ class SimParams final : public ParamsDataInterface<SimParamsData>
         // Construct with imported data and max field substeps
         static Input from_import(ImportData const&,
                                  SPConstParticles,
+                                 size_type max_steps,
```

Comment:
Maybe we should pass `inp::TrackingLimits const&` instead of this and field substeps?

Reply:
Done, and updated the `TrackingLimits` max field substeps to match the driver options.